addon/components/frost-bunsen-input-select/component.js

```diff
@@ -0,0 +1,68 @@
+import _ from 'lodash'
+import Ember from 'ember'
+import computed, {readOnly} from 'ember-computed-decorators'
+
+import InputMixin from 'ember-frost-bunsen/mixins/input'
+import ListInputMixin from 'ember-frost-bunsen/mixins/list-input'
+import layout from './template'
+
+export default Ember.Component.extend(InputMixin, ListInputMixin, {
```

Comment:
There will be only the one. this will become `BaseInput.extend(ListInputMixin...`